addons/website_sale/views/templates.xml

```diff
@@ -1105,6 +1108,18 @@
                                         </div>
                                         <div id="product_option_block" class="d-flex flex-wrap w-100"/>
                                     </div>
+                                    <div t-if="accordion_enabled" id="product_accordion" class="o_accordion_not_initialized accordion accordion-flush mt-4" >
+                                        <div t-if="accordion_custom_item" class="accordion-item">
+                                            <h2 class="accordion-header">
+                                                <button class="accordion-button collapsed fw-medium" type="button" data-bs-toggle="collapse" data-bs-target="#customisable_item" aria-expanded="false" aria-controls="customisable_item">
+                                                    Customizable accordion item
+                                                </button>
+                                            </h2>
+                                            <div id="customisable_item" class="accordion-collapse collapse" data-bs-parent="#product_accordion">
+                                                <div class="accordion-body pt-0">This accordion item is customizable and shared between all products.</div>
```

Comment:
Unfortunately the editor doesn't know how to correctly treat your custom component :/

https://github.com/user-attachments/assets/4e2dde59-c7ec-4991-b886-fe9e1882541d

I suspect that's due to the `<h2>` that's recognised as a `drop-near` zone. You may want to refactor the code, replacing `h2` ... + other eventual changes. The goal is to:
- allow inner snippets to be dropped into the entry
- avoid that other snippets (the big ones) could be dropped into the entry
- avoid that the entry could be removed/moved

I made a try using the DOM of the collapse snippet and seem to work fine (except that then element can be removed...  and we don't want that!). For reference:

```xml
<section t-if="accordion_enabled" class="s_faq_collapse pt32 pb32 s_faq_collapse_light oe_unremovable oe_unmovable " data-snippet="s_faq_collapse" data-name="Accordion">
    <div class="container">
        <div t-if="accordion_custom_item" id="myCollapse" class="accordion oe_unremovable oe_unmovable " role="tablist">
            <div class="card bg-white" data-name="Item" role="presentation">
                <a href="#" role="tab" data-bs-toggle="collapse" aria-expanded="true" class="card-header o_default_snippet_text" data-bs-target="#myCollapseTab879939_1" aria-controls="myCollapseTab879939_1">Customizable accordion item</a>
                <div class="collapse show" data-bs-parent="#myCollapse" role="tabpanel" id="myCollapseTab879939_1">
                    <div class="card-body">
                        <p class="card-text o_default_snippet_text">These terms of service ("Terms", "Agreement") are an agreement between the website ("Website operator", "us", "we" or "our") and you ("User", "you" or "your"). This Agreement sets forth the general terms and conditions of your use of this website and any of its products or services (collectively, "Website" or "Services").</p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>
```

I think that using the default snippet makes sense (if feasible)... you may want to get in touch with @Brieuc-brd, he is struggling  with this snippet in this very moment.

If implementing the "custom entry" takes to much time let's skip it, we'll do it later